lib/phx_tools_web/plugs/curl_detector.ex

```diff
@@ -0,0 +1,44 @@
+defmodule PhxToolsWeb.CurlDetector do
+  @moduledoc """
+  A plug that serves a script based on the request's `User-Agent` or specific URL paths.
+
+  Requests from `curl` receive the script content directly, halting further processing. Additionally, requests to `/Linux.sh` or `/macOS.sh` paths return the script regardless of `User-Agent`.
```

Comment:
These two files are not part of the plug anymore, so I'd either remove their mention or change the wording to make it clear this plug is not responsible for them and they are just regular files.

Reply:
Right away